packages/maskbook/src/extension/background-script/WelcomeServices/generateBackupJSON.ts

```diff
@@ -93,6 +100,29 @@ export async function generateBackupJSON(opts: Partial<BackupOptions> = {}): Pro
         const wallets_ = (await getWallets(ProviderType.Maskbook)).map(WalletRecordToJSONFormat)
         wallets.push(...wallets_)
     }
+
+    async function backupAllPlugins() {
+        await Promise.all(
+            [...activatedPluginsWorker]
+                // generate backup
+                .map(async (plugin) => {
+                    const f = plugin.backup?.onBackup
+                    if (!f) return
+
+                    async function backupPlugin() {
+                        const object = await timeout(f!(), 3000)
```

Comment:
You meant this, right?
```suggestion
                        const object = await timeout(f!, 3000)
```

Reply:
Nope, but I have renamed those variables to clearify.